src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -351,9 +352,10 @@ export default class ArrayField extends React.Component {
                     />
                   </div>
                   <button
+                    type="button"
                     className="usa-button-secondary edit vads-u-flex--auto"
+                    aria-label={`Edit ${itemName}`}
                     onClick={() => this.handleEdit(index)}
-                    aria-label={`Edit ${title}`}
```

Comment:
Would it be better if we had `itemName` default to `title` instead of "Item" for the Edit and Update buttons?

Reply:
I'm not sure if that is a good idea because the page title doesn't always match the item... e.g. for the prisoner of war page confinement ranges, the title is undefined, but we do define an item name.